en_US/installation/0-setup.dns.md

```diff
@@ -9,14 +9,14 @@ __IMPORTANT NOTE__: `A`, `MX` records are required, `Reverse PTR`, `SPF`,
 
 ### What is an `A` record
 
-`A` records map a FQDN (fully qualified domain name) to an IP address. This is
+`A` records map a FQDN (fully qualified domain name) to an IP(v4) address. This is
```

Comment:
`A` record could be an IP v6 address, so this change is not right. :)

Reply:
I mean the `AAAA` record is for ipv6 right?